### PR TITLE
Add documentation for GA4 product_download event

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Add documentation for GA4 product_download event.

## Issue / Bugzilla link

#13238 

## Testing

`make livedocs`

http://127.0.0.1:8100/attribution/0001-analytics.html#product-download

### CODE REVIEW

File: docs/attribution/0001-analytics.rst [hunk 1]
```diff
@@ -283,6 +283,52 @@ To use this method you will need to include ``datalayer-begincheckout-init.es6.j
...
 
 
+Product Download
+~~~~~~~~~~~~~~~~
+
+.. Important::
+
+    Only Firefox and Pocket are currently supported. VPN support has not been added.
+
+We are using a the custom event `product_download` to track product downloads (Firefox and VPN). We are not using
```
Comment (alexgibson) [inline docs/attribution/0001-analytics.rst:293]: I'm a little confused by these opening lines, it says only Firefox and Pocket are supported, but the next line says Firefox and VPN? Which is right?

Also, by Pocket, do you mean in pocket mode? (if so, should that be covered in the pocket mode section?)

**Review by alexgibson — APPROVED** [review]
Looks good! just one question but otherwise r+wc

File: docs/attribution/0001-analytics.rst [hunk 2]
```diff
@@ -283,6 +283,52 @@ To use this method you will need to include ``datalayer-begincheckout-init.es6.j
...
+the default GA4 event file_download for a combination of reasons: it does not trigger for the Firefox file types,
+we would like to collect more information than is included with the default events, and we would like to treat
+product downloads as goals but not all file downloads are goals.
+
+Links to the app stores directly or through adjust are also treated as product downloads.
+
+.. Note::
+
+    Most apps listed in *appstores.py* are supported but you may still want to check that the URL
+    you are tracking is identified as valid in `isValidDownloadURL` and will be recognized by `getEventFromUrl`.
```
Comment (alexgibson) [inline docs/attribution/0001-analytics.rst:303]: Should these event and function names all be in double back-ticks for them to be formatted using `<code>`?
```suggestion
    you are tracking is identified as valid in ``isValidDownloadURL`` and will be recognized by ``getEventFromUrl``.
```

File: docs/attribution/0001-analytics.rst (hunk 1 above)
Comment (stephaniehobson) [reply docs/attribution/0001-analytics.rst]: Clear now?

Comment (alexgibson) [reply docs/attribution/0001-analytics.rst]: It doesn't yet seem super clear to me where we are and aren't using it from the docs, but I'm guessing this is also documentation for work that's still in progress, so let's move forward and update as we go? :)

**Pull request merged**